apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -39,6 +41,14 @@ export const ConnectWalletButton: FC<
         address={address}
         dataLayoutId={dataLayoutId}
         className={className}
+        balance={Object.values(Asset).map(asset => (
```

Comment:
I don't like this approach, you should rather pass the whole component there and I believe that the component should be in the frontend app, not in the UI package.